### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (kbd:nth-child(1))

### ISSUE THREAD

**Issue by WilliamBerryiii** [microsoft/hve-core#1907]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[HVE Core](https://microsoft.github.io/hve-core/)
#### Element path
kbd:nth-child(1)
#### Snippet
`<kbd class="searchHint_iIMx">ctrl</kbd>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 1.51 (foreground color: #bec3c9, background color: #ebedf0, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 149.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
